seacrowd/sea_datasets/filipino_gay_lang/filipino_gay_lang.py

```diff
@@ -0,0 +1,115 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import SCHEMA_TO_FEATURES, TASK_TO_SCHEMA, Tasks
+
+# TODO: Add BibTeX citation
+_CITATION = """\
+@article{oco2015witchebelles,
+  author    = {Oco, Nathaniel and Fajutagana, Raymart and Lim, Christine Mae and Mi{\~n}on, Judi Diane and Morano, Julie-Ann and Tinoco, Ryan Christian},
```

Comment:
Invalid escape sequence `\~`. My suggestion is to convert `_CITATION` into a raw string by adding an `r` prefix (e.g., `_CITATION = r"""`)